packages/medusa-core-utils/yarn.lock

```diff
@@ -559,6 +559,20 @@
     "@babel/helper-split-export-declaration" "^7.8.3"
     globals "^11.1.0"
 
+"@babel/plugin-transform-classes@^7.9.5":
```

Comment:
Delete `.lock` file and add to ignore

Reply:
https://stackoverflow.com/questions/44552348/should-i-commit-yarn-lock-and-package-lock-json-files/44904494

Reply:
Alright.. Thought it was like package lock.

But then it adds 5000-10.000 lines of code for every package we have. I was just thinking whether or not this could be done differently